hostinfo/misc.lua

```diff
@@ -132,5 +135,24 @@ local function asyncSpawn(dataArr, spawnFunc, successFunc, finalCb)
   end)
 end
 
+local function logWarn(errTable)
+  local logpath
+  if los.type() ~= 'linux' then
+    -- TODO: properly implement logging in windows
+    logpath = './'
+  else
+    logpath = '/var/log'
+  end
+  logpath = join(logpath, 'rackspace-monitoring-agent-hostinfo.log')
+  logging.init(logging.StdoutFileLogger:new({
```

Comment:
The file that contains your line there(main.lua) inits the logger with a logpath derived from the -l argument to the cli. 
```
  -- Setup Logging
  logging.init(logging.StdoutFileLogger:new({
    log_level = log_level,
    path = argv.args.l
  }))
```
Trying to run the logger without initing it with a  logpath doesnt work. (missing lib err || missing numeric literal at line 1 column 6).
Trying to run logging.logf without initing the logger results in the following err:
```
Uncaught exception:
[string "bundle:deps/logging/init.lua"]:92: attempt to compare number with nil
```